### ISSUE TITLE
pachyderm-aml-syncer.service is not enabled on startup

### ISSUE THREAD

**Issue by RaananHadar** [pachyderm/aml#65]
I powered down the syncer VM when not using the preview. When turning it back on the pachyderm-aml-syncer.service was not running. It appears it is not enabled by default.

Could be useful to enable it by default if the VM will crash for whatever reason.

**Comment by albscui** [comment]
> @albscui is there a reason we dont `systemctl enable pachyderm-aml-syncer.service` so it auto starts?

Yes, during the initial setup, the Syncer depends on a set of environment variables to know which Azure ML workspace to connect to. We first copy these variables to the VM, then run the syncer as a systemd service. However, if the user shutsdown the VM, then the user would have to manually start the systemd service upon starting the VM again. Thanks @RaananHadar for pointing this out.

We could update the syncer to crash loop or wait for the environment variables to be set. Then we can configure this systemd service to start automatically.
